docs/orchestration/kubernetes/minio-distributed-statefulset.yaml

```diff
@@ -24,10 +24,7 @@ spec:
         image: minio/minio:RELEASE.2019-08-29T00-25-01Z
         args:
         - server
-        - http://minio-0.minio.default.svc.cluster.local/data
-        - http://minio-1.minio.default.svc.cluster.local/data
-        - http://minio-2.minio.default.svc.cluster.local/data
-        - http://minio-3.minio.default.svc.cluster.local/data
+        - http://minio-{0...1}.minio.default.svc.cluster.local/data
```

Comment:
```suggestion
        - http://minio-{0...3}.minio.default.svc.cluster.local/data
```

Reply:
This should be `minio-{0...3}`, looks like didn't get updated